### DISCUSSION TITLE
奇奇怪怪的提问和回答

### DISCUSSION THREAD

**Discussion by kaokao221** [ClassIsland/ClassIsland#66] [常规]
问一下现在（一个小时前）closed的issues是在源码中实现并且在清明后实装到Beta版本还是正式版本呢？清明节前能否把closed issues都上到正式版本啊，另外集控什么时候能成型？

**Comment by HelloWRC** [comment]
> 问一下现在（一个小时前）closed的issues是在源码中实现并且在清明后实装到Beta版本还是正式版本呢？清明节前能否把closed issues都上到正式版本啊，另外集控什么时候能成型？

理论上目前这些功能可以在清明后下放到测试版。如果您想要体验最新的构建，可以在[AppVeyor](https://ci.appveyor.com/project/HelloWRC/classisland/build/artifacts)上下载，这些版本是直接从最新的代码构建的，可能会不稳定。

正式版要等把1.4所有开发目标实现后（[链接](https://github.com/users/HelloWRC/projects/2/views/1)）才会发布正式版。

预计这个测试版发布之后，就会开始继续开发集控功能。不过由于我时间有限，集控相关功能可能要比较久的时间才能完全开发好。
